packages/resin-event-log/src/adaptors/ga.js

```diff
@@ -0,0 +1,28 @@
+var ResinGaClient = require('resin-universal-ga')
+
+module.exports = function (options) {
+	var debug = options.debug,
+		propertyId = options.gaId,
+		site = options.gaSite
+
+	if (!(propertyId && site)) {
+		if (debug) {
+			console.warn("`gaId` and/or `gaSite` are not set, GA tracking is disabled")
+		}
+		return null
+	}
+
+	var gaClient = ResinGaClient(propertyId, site, debug)
+
+	return {
+		login: function(user) {
+			return gaClient.login(user.id)
+		},
+		logout: function() {
+			return gaClient.logout()
+		},
+		track: function (prefix, type, data) {
+			return gaClient.track(site, type, prefix)
```

Comment:
Yes, the param is here to indicate that the adaptor interface is uniform but GA doesn't allow extra data payload. Will put a comment around this param.